cmd/utils_test.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import "testing"
+
+func Test_validateName(t *testing.T) {
+	type args struct {
+		name string
+	}
+	const errorString string = "a DNS-1123 label must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name',  or '123-abc', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?')"
+
+	tests := []struct {
+		testCase string
+		name     string
+		error    string
```

Comment:
why do we need to store error string?
It is ok to just check if there is an error or not. We don't need to check error string.